api/controllers/media.js

```diff
@@ -1,11 +1,14 @@
 const uuidv1 = require('uuid/v1');
 const azure = require('azure-storage');
-const blobService = azure.createBlobService(process.env.AZURE_STORAGE_CONNECTION_STRING);
+const blobService = azure.createBlobService(
+  process.env.AZURE_STORAGE_CONNECTION_STRING
+);
 
 const BLOB_CONTAINER_NAME = process.env.BLOB_CONTAINER_NAME || 'cblob';
 
 module.exports = {
-  uploadMedia: uploadMedia
+  uploadMedia,
+  uploadAnalytics
```

Comment:
to me , analytics must be separate from media controller. it needs an own controller

Reply:
Seperated